ramp.md

```diff
@@ -14,31 +14,38 @@ or you cannot get feedback from any professors or labs that you have already wor
 
 **If you are an EPFL Master's student**:
 Before contacting people below, ask members of the lab(s) you're working with for your semester project/thesis.
-They will be able to give the most relevant feedback. If you cannot get feedback from your lab, you are encouraged to contact the people below.
+They will be able to give the most relevant feedback. If you cannot get feedback from your lab, you are encouraged to apply for the program.
 
 
 ## Who is this by?
 
 Some PhD students in the Computer & Communication Sciences department
 at EPFL are volunteering to help review your application materials. This 
-initiative is called EPIC RAMP, and has started its inaugural iteration 
-for the 2022 application cycle.
+initiative is called EPIC RAMP, and it has entered its second cycle in 2023.
 
 Please understand that this is a service that the PhD students are providing
 in their free time, so it may take a few weeks to get a response, and we may
-have to pause the program if we get too many requests. We may refuse to help
+have to pause the program if we get too many requests. Also, 
+please note that RAMP operates independently of EDIC's admission committee
+and it is not considered a component of the admission process.
+
+<!-- We may refuse to help
 people who mass email our list of volunteers instead of choosing one or two,
-or who do not take the time to first follow the guides on this website.
+or who do not take the time to first follow the guides on this website. -->
 
 
 ## Steps
 
 1. Follow [the Applying section of the EPIC PhD guide](/applying) for statements of purpose and resumes
-2. Find _one or two_ people in the list [below](#volunteers) whose research interest match yours
-3. Send them an email introducing yourself and attaching your statement of purpose and resume (we provide a template [below](#email-template))
+2. Apply for RAMP by filling the relevant information [here](https://forms.gle/xSpr3hipj6RBCY7w8)
+3. We will get back to you in a few days and let you know of your mentor
+4. Get in touch with your mentor and share with them your Statement of Purpose and Resume
```

Comment:
`Reach out to your mentor with any questions you have, attaching your statement of purpose and resumè so they can best assist you`

Reply:
addressed the comment in the next commit